packages/react-icons/bin/index.js

```diff
@@ -7,24 +7,39 @@ const {rmSync} = require('fs-extra');
 
 const iconsSourceDirPath = 'node_modules/@coveord/plasma-tokens/icons';
 
-const convertIcon = async ([iconName, variants]) =>
-    Promise.all([
-        fs.appendFile('./generated/index.ts', `export * from './${iconName}';\n`),
+const findIconName = (filename) => filename.replace(iconsSourceDirPath, '').substring(1).split('/')[0];
+const findVariantName = (filename) => filename.replace(iconsSourceDirPath, '').split('/').pop().replace('.svg', '');
+const getComponentName = (file) => {
+    const iconName = findIconName(file);
+    const variantName = findVariantName(file);
+    return `${upperFirst(iconName)}${upperFirst(variantName)}`;
+};
+
+const convertIcon = async ([iconName, variants]) => {
+    await fs.ensureDir(`./generated/${iconName}`);
+    return Promise.all([
         ...variants.map(convertVariant),
+        fs.appendFile('./generated/index.ts', `export * from './${iconName}';\n`),
     ]);
+};
 
 const convertVariant = async (file) => {
     try {
         const iconName = findIconName(file);
         const variantName = findVariantName(file);
-        const componentName = `${upperFirst(iconName)}${upperFirst(variantName)}`;
+        const componentName = getComponentName(file);
         const fileContent = await fs.readFile(file);
         const tsCode = await svgr.transform(
             fileContent.toString('utf8'),
-            {typescript: true, exportType: 'named', namedExport: componentName},
+            {
+                typescript: true,
+                exportType: 'named',
+                namedExport: componentName,
+                svgProps: {className: '{"icon mod-stroke " + props.className}', },
```

Comment:
This means all icons now have `icon` `mod-stroke` classes by default

Reply:
won't this create conflicts with the style if we add mod-fill? :eyes:

Reply:
`mod-fill`? this class doesn't exist 🤔